prql-python/python/tests/test_all.py

```diff
@@ -0,0 +1,21 @@
+import prql_python as prql
+import json
+
+
+def test_all():
+    """
+    Test the basic python functions
+    """
+
+    prql_query = "from employee"
+
+    # Since the AST is so in flux lets just take these dont throw exceptions
+    res = json.loads(prql.to_json(prql_query))
+    assert res is not None
+
+    res = prql.to_sql(prql_query)
+    assert res is not None
+
+
+if __name__ == '__main__':
```

Comment:
I haven't used `nox` much, but traditionally in pytest, this isn't required — pytest will collect and run the tests

Reply:
Yep, the `if` statement is not required for nox. All it's doing is running purest manually in isolated virtualenvs of different versions.